src/main/java/org/spongepowered/api/text/format/TextStyle.java

```diff
@@ -178,18 +188,21 @@ public TextStyle negate() {
      * @return A new text style composed out of the given text styles
      */
     public TextStyle and(TextStyle... styles) {
+        return and(styles, false);
+    }
+
+    private TextStyle and(TextStyle[] styles, boolean negate) {
```

Comment:
this method with the boolean negate is not only unhelpful, it makes the intention less clear